{
  try {
    proc.waitForStop();
    assertFalse("Expected process to fail",fails);
  }
 catch (  ExecException e) {
    assertTrue("Unexpected failure in worker process",fails);
  }
}
